test/services/PenaltiesServiceSpec.scala

```diff
@@ -44,11 +50,25 @@ class PenaltiesServiceSpec extends AnyWordSpecLike with MockFactory with Matcher
     new PenaltiesService(mockPenaltiesConnector)
   }
 
-  "Calling getPenaltiesDataForVRN" should {
+  def penaltiesServiceWithoutMock() : PenaltiesService = {
```

Comment:
Sorry to nitpick, but you don't need these 2 versions of the service. You can just have `def penaltiesService(): PenaltiesService = new PenaltiesService(mockPenaltiesConnector)` and then call `setup()` only in the test that needs the mock